app/classifier/tasks/drawing/markings-renderer.cjsx

```diff
@@ -18,9 +18,8 @@ module.exports = React.createClass
     # console.log 'Old marks were', @state.oldSetOfMarks
     newSetOfMarks = []
     # Automatically select new marks.
-    annotations = nextProps.classification?.annotations ? []
-    annotation = annotations[annotations.length - 1]
-    if annotation?
+    annotation = nextProps.annotation
```

Comment:
Thanks. I was more wondering if this still works with drawing tasks, which are presumably passing annotations in via the classification. So I was asking if this change is backwards compatible with the existing code.